Assets/Scripts/TodoList.cs

```diff
@@ -116,10 +111,15 @@ void FillImage(Image img)
             //added to coordinate with pick up spawner
             left[nextObject].tag = "PointsPickUp"; //sets tag
             left[nextObject].IsThisOBJForPoints = true; //set true for points
+        } else
+        {
+            Debug.Log("tried to load object that doesnt exist");
         }
-        FindObjectOfType<PickUpableSpawner>().FindOBJ();
+
+        FindObjectOfType<PickUpableSpawner>().FindOBJ(); //called to coordinate something with pickupablespawner
     }
 
+    //DIDNT DELETE THIS because it could be used in the future
```

Comment:
I don't  care remove it, we can always do some git time travel. large blocks of commented out code make reading the file more difficult. Additionally, pickUp has already been changed on master.

Reply:
Brandons highlighting can be put right there if he uncomments it